pkg/lib/context-menu.css

```diff
@@ -0,0 +1,22 @@
+.contextMenu {
+    position: fixed;
+    z-index: 101;
```

Comment:
This could use a comment: Both why it needs to be set, and explain the magic number.